install/autodetect.go

```diff
@@ -173,10 +173,17 @@ func (k *K8sInstaller) autodetectAndValidate(ctx context.Context) error {
 }
 
 func (k *K8sInstaller) autodetectKubeProxy(ctx context.Context) error {
+	if k.params.UserSetKubeProxyReplacement {
+		return nil
+	} else if k.flavor.Kind == k8s.KindK3s {
```

Comment:
@brb 
Sorry I could have mentioned it. I initially thought to detect k3s cluster using annotations but later I found that we can use `k.flavor.Kind` .